isofit/core/forward.py

```diff
@@ -230,7 +230,7 @@ def calc_rdn(self, x, geom, rfl=None, Ls=None):
 
         rfl_hi = self.upsample(self.surface.wl, rfl)
         Ls_hi = self.upsample(self.surface.wl, Ls)
-        return self.RT.calc_rdn(x_RT, rfl_hi, Ls_hi, geom)
```

Comment:
@unbohn ; I inherited this update from your changes...but I think this is actually wrong?  Wouldn't it break the wl_shift case?  If the wavelength grids match, it's irrelevant.

Reply:
@pgbrodrick In the wl_shift case, this element is part of `x_instrument`, right? So, I think passing `x_surface` to `self.RT.calc_rdn` wouldn't break anything as it's only used for accessing the last and second last index of the surface state in the glint_model case.

Reply:
Nice.  I confirmed this works with a test as well.